### DISCUSSION TITLE
Expression Set Version and Context Definitions not included in SGD

### DISCUSSION THREAD

**Discussion by PythSomeone** [scolladon/sfdx-git-delta#958] [Q&A]
Hi,

When a delta is generated, changes or creations of the Expression Set Version and Context Definition are not shown in the resulting delta.

Could you please add support for both types of metadata?

**Comment by scolladon** [comment]
Hi @PythSomeone !

Thanks for raising this issue and thanks for contributing in making this project better!

Yes sure !
Just to be sure, you want those metadata available right ?
- [ExpressionSetDefinition](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_expressionsetdefinition.htm) (already there)
- [ContextDefinition](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_contextdefinition.htm?q=Context%20Definitions)

Is there other metadata from the same area ?

**Reply by PythSomeone** [reply]
Hi @scolladon 

ExpressionSetDefinition is detected by Delta just fine, the problem appears to be with ExpressionSetDefinitionVersion which is not present in the delta output.

**Reply by PythSomeone** [reply]
@scolladon 
I also noticed that ProductSpecificationRecType and ProductSpecificationType were also not included in the generated delta, could you please take a look at that?

**Reply by scolladon** [reply]
Sure @PythSomeone 
I'll do it in the afternoon ?
Could you have a look at it [here](https://github.com/scolladon/sfdx-git-delta/pull/960) and let us know please ?

**Reply by PythSomeone** [reply]
@scolladon ExpressionSetDefinitionVersion is still not included, left the comment in the issue
